src/simulations/simulations.jl

```diff
@@ -3,33 +3,20 @@ import DiffEqBase: solve
 
 """
 ```Julia
-Perturbation(;node,var,f)
+ChangeInitialConditions(;node,var,f)
```

Comment:
should the `ChangeInitialConditions` maybe also get a separate file in faults/ ?

Reply:
We can do that. There is no simulations.jl file there anymore then.